src/main/java/org/jenkinsci/test/acceptance/po/Folder.java

```diff
@@ -58,4 +69,67 @@ public JobsMixIn getJobs() {
     public ViewsMixIn getViews() {
         return views;
     }
+
+    @Override
+    public void delete() {
+        open();
+        clickLink("Delete Folder");
+        waitFor(by.button("Yes")).click();
+    }
+
+    public void setEnvironmentalVariables(final Map<String, String> envVbles) {
+        configure();
```

Comment:
Not sure what you mean @olivergondza 

Do you mean configure and save should be called from the outside? Or that save should not be called?
I made it that way so you can do folder.setEnvironmentalVariables(xxx) without opening configuration and saving.